jplag.frontend-utils/src/main/java/de/jplag/TokenPrinter.java

```diff
@@ -119,12 +119,42 @@ public static String printTokens(TokenList tokens, Collection<File> files, File
         return builder.toString();
     }
 
+    private static StringBuilder appendCodeLine(StringBuilder builder, String currentLine) {
+        if (REPLACE_TABS) {
+            currentLine = currentLine.replaceAll(TAB, TAB_REPLACEMENT);
+        }
+        return builder.append(currentLine);
+    }
+
+    /**
+     * Adds padding to the given StringBuilder in order to reach the targetPosition.
+     */
+    private static int addPadding(StringBuilder builder, int currentPosition, String currentLine, int targetPosition) {
```

Comment:
Could you provide more information on the parameters :) Do you have information regarding the index of "currentPosition" are these indices within the currentLine? Indexed by 0 ? Is that clear to you? If so, please add it to the docs .. this will help future development

Reply:
I agree. I hope the new documentation is more helpful.